R/tar-stars.R

```diff
@@ -0,0 +1,136 @@
+#' Create a stars _stars_ Target
+#'
+#' Provides a target format for stars objects.
+#'
+#' @param driver character. File format expressed as GDAL driver names passed to [stars::write_stars()]. See [sf::st_drivers()].
+#' @param options character. GDAL driver specific datasource creation options passed to [stars::write_stars()]
+#' @param ... Additional arguments not yet used
+#'
+#' @inheritParams targets::tar_target
+#' @importFrom rlang %||% arg_match0 is_installed
```

Comment:
We don't need to use `#' @importFrom rlang <fun>` each time we want to use those functions - I suggest that we move these to https://github.com/njtierney/geotargets/blob/master/R/geotargets-package.R#L4-L5

Reply:
done in https://github.com/njtierney/geotargets/pull/33/commits/0e20c172f574efed855625ec06219c40eade9917 and https://github.com/njtierney/geotargets/pull/33/commits/78bdee5347f99818837669bac4aa75e69911f6f4